welltestpy/data/testslib.py

```diff
@@ -467,6 +467,27 @@ def plot(self, wells, exclude=None, fig=None, ax=None, **kwargs):
             **kwargs
         )
 
+    def diagnostic_plot(self, observation, **kwargs):
```

Comment:
observation should be the name of the observation well. You can get the observation itself afterwards by:
```python
def diagnostic_plot(self, well, **kwargs):
    observation = self.observations[well]
    ...
```